cpp/daal/src/algorithms/pca/transform/oneapi/pca_transform_dense_default_batch_oneapi_impl.i

```diff
@@ -138,7 +149,7 @@ services::Status TransformKernelOneAPI<algorithmFPType, method>::whitening(Execu
     KernelPtr kernel                   = factory.getKernel(whiteningKernel, status);
```

Comment:
Add check status for buildKernel few lines above.

Reply:
fixed